[DOCS] Proofreading Dev Guide in DL Streamer docs (#897)
open-edge-platform/edge-ai-libraries: libraries/dl-streamer/docs/source/dev_guide/coding_style.md

# Coding Style

> We use Clang formatter for our code. Code style checker is being run

Reviewer: ```suggestion
Because Clang formatter is used for code, style checker executed 
```

> Because Clang formatter is used, style checker executed 
> through CI, ensure correct code formatting before merging it to `master`.
> There is a configuration file for ``clang-format-7`` 
> (`.clang-format` in the root folder of this project).
> You will need to install it locally and configure your IDE to format
> your code with it.

Reviewer: ```suggestion
Before merging changes to `master`, make sure the code is formatted properly.
Use the Clang formatter, executed through CI, for this purpose.
You will need to install it locally and configure your IDE to format
your code with it. You can use a configuration file for ``clang-format-7``  
from the root of this project (`.clang-format`).
```
